sw/airborne/modules/gps/gps_ubx.c

```diff
@@ -86,6 +86,11 @@ extern struct RtcmMan rtcm_man;
 #ifndef INJECT_BUFF_SIZE
 #define INJECT_BUFF_SIZE 1024 + 6
 #endif
+
+#ifndef GPS_UBX_BOOTRESET
+#define GPS_UBX_BOOTRESET 0
+#endif
+
```

Comment:
this part is wrongly placed inside the `USE_GPS_UBX_RTCM` section, you should move it either before or after.

Reply:
Oh, sorry for the mistake. I'll fix it in a moment.